episodes/08-putting-it-all-together.md

```diff
@@ -146,7 +146,7 @@ for the rest.
 
 We will cover a few basic commands for creating and formatting plots with matplotlib in this lesson.
 A great resource for help creating and styling your figures is the matplotlib gallery
-([http://matplotlib.org/gallery.html](https://matplotlib.org/gallery.html)), which includes plots in many different
+([[http://matplotlib.org/gallery.html](https://matplotlib.org/stable/gallery/index.html)](https://matplotlib.org/stable/gallery/index.html)), which includes plots in many different
```

Comment:
I wonder if this one should be:
```md
the [matplot gallery](https://matplotlib.org/stable/gallery/index.html), which includes ...
```

Reply:
I think these are important link changes that are in limbo.  Will try to resolve this pull request with minor edits.